src/components/forms/ProjectDetailsForm.tsx

```diff
@@ -31,6 +32,8 @@ export default function ProjectDetailsForm({
   loading?: boolean
   onValuesChange?: VoidFunction
 }) {
+  const initialLogoUrl = useWatch('logoUri', form)
```

Comment:
Also noticed sometimes the logo wasn't loading at all and it appeared like this on initial load:

<img width="1114" alt="Screen Shot 2023-01-16 at 3 08 19 pm" src="https://user-images.githubusercontent.com/96150256/212602882-7b3888b0-c14a-484b-8ef7-f2a5175f66c4.png">

This fixes that.